### PR TITLE
add travel history into fhir converter and ecr viewer

### LINKED ISSUE #1001: Extract patient travel history from eICR; add to eCR Viewer
## Action Requested
Add patient travel history to the eCR Viewer (currently it's stubbed).

## Acceptance Criteria
* When eCRs go through conversion to FHIR, patient travel history is extracted (if present)
* When eCRs are shown in the eCR Viewer, patient travel history is shown according to the designs (if present)

## Testing
We'll need some sample eCRs to test with that include travel history.

## Background & Context
The eCR Viewer designs call for including patient travel history, but it's not currently included in our FHIR conversion process.

## Additional Information
This is likely the file that will need to be updated within the converter: [SocialHistory/_entry.liquid](https://github.com/CDCgov/phdi/blob/main/containers/fhir-converter/Templates/eCR/Entry/SocialHistory/_entry.liquid)
And this file within the eCR Viewer will need the travel history FHIR path added: [fhirPath.yml](https://github.com/CDCgov/phdi/blob/main/containers/ecr-viewer/public/fhirPath.yml)

We will need to figure out where in the eCR CCDA spec patient travel history is found. The [FHIR implementation guide](https://build.fhir.org/ig/HL7/case-reporting/StructureDefinition-eicr-composition-definitions.html#diff_Composition.section:sliceSocialHistorySection.entry:sliceTravelHistory) for eICR composition may be helpful here.

### PR DESCRIPTION
**Pull request by BobanL**
# PULL REQUEST

## Summary
- Add new observation template for travel history
- Add travel history to the ecr viewer
- Add unit tests for travel history
 
## Related Issue
Fixes #1001

## Additional Information
- This currently works with the limited travel history we are given, but this may need more refactor in the future depending on how other examples of travel history look.

image

### CODE REVIEW

**Review by KennethSkylight — CHANGES_REQUESTED** [review]
Just a few changes

**Review by KennethSkylight — APPROVED** [review]
lgtm!

File: containers/ecr-viewer/seed-scripts/fhir_data/1dd10047-2207-4eac-a993-0f706c88be5d.json [hunk 1]
```diff
@@ -238,7 +238,12 @@
                 }
```
Comment (emmastephenson) [inline containers/ecr-viewer/seed-scripts/fhir_data/1dd10047-2207-4eac-a993-0f706c88be5d.json:238]: Nice call on updating some of the seed data 👍

File: containers/ecr-viewer/src/app/tests/utils.test.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  it("should have no available data when there is no data", () => {
+    const actual = evaluateSocialData(undefined, mappings);
+    expect(actual.available_data).toBeEmpty();
+    expect(actual.unavailable_data).not.toBeEmpty();
+  });
+  it("should have travel history when there is a travel history observation present", () => {
+    const bundleWithTravelHistory = {
+      resourceType: "Bundle",
+      type: "batch",
+      entry: [
```
Comment (emmastephenson) [inline containers/ecr-viewer/src/app/tests/utils.test.tsx:16]: Out of scope for this PR, but should we have an assets folder or something to store the test bundles? We have one in `phdi` ([/tests/assets/](https://github.com/CDCgov/phdi/tree/main/tests/assets)) but maybe it's time to add container-specific assets too.

**Review by emmastephenson — APPROVED** [review]
Looks great! Thanks for figuring this one out!

File: containers/ecr-viewer/src/app/tests/utils.test.tsx (hunk 2 above)
Comment (BobanL) [reply containers/ecr-viewer/src/app/tests/utils.test.tsx:16]: I'll make a follow up pr to address this. I'll just put it in the tests folder.

**Pull request merged**